generators/chipyard/src/main/scala/TestHarness.scala

```diff
@@ -17,13 +17,14 @@ import chipyard.clocking.{SimplePllConfiguration, ClockDividerN}
 // -------------------------------
 
 case object BuildTop extends Field[Parameters => LazyModule]((p: Parameters) => new ChipTop()(p))
-
-trait HasTestHarnessFunctions {
-  val harnessFunctions = ArrayBuffer.empty[HasHarnessSignalReferences => Seq[Any]]
-}
+case object DefaultClockFrequencyKey extends Field[Double](100.0)
```

Comment:
I would like this to be commented or at least named mentioning that this is in MHz